DabraWarq/DWdavies01.xml

```diff
@@ -9,7 +9,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <editor role="cataloguer" key="GS"/>
         <editor role="generalEditor" key="AB"/>
       </titleStmt>
-
+      <editionStmt>
+        <ab> This lavish seventeenth manuscript is preserved in the Dabra Warq Monastery, Goǧǧām,
+          Ethiopia. It was microfilmed by Donald Davies between 1960–1970. Full images (balck and
```

Comment:
manuscript identifiers should be listed as altIdentifier

identifiers of digital copies should be listed as surrogates https://betamasaheft.eu/Guidelines/?q=surrogates&id=surrogates

other catalogues should be linked as relation

if you used PEMM for your description this should be mentioned in the change element

editionStmt is about **THIS** edition, this whole text must be moved away from here to the relevant sections

Reply:
This is fine but the records should still be linked and the other corrections are also needed